ui/src/message_popup/author/question_button.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import {withLearningObjectiveAndIndicator} from '../transformations.jsx';
+import * as Routes from '../../routes.js';
+
+import {ListItem} from 'material-ui/List';
+
+import SchoolIcon from 'material-ui/svg-icons/social/school';
+
+export default React.createClass({
+  displayName: 'QuestionButton',
+
+  propTypes: {
+    question: React.PropTypes.object.isRequired
+  }, 
+
+  changePath(){
```

Comment:
I think I probably missed this earlier, but please use the convention where event handlers start with `on` and indicate what event occurred, so something like `onItemClicked`.